wpilibNewCommands/src/main/native/include/frc2/command/ParallelCommandGroup.h

```diff
@@ -50,11 +54,9 @@ class ParallelCommandGroup
    *
    * @param commands the commands to include in this composition.
    */
-  template <class... Types,
-            typename = std::enable_if_t<std::conjunction_v<
-                std::is_base_of<Command, std::remove_reference_t<Types>>...>>>
-  explicit ParallelCommandGroup(Types&&... commands) {
-    AddCommands(std::forward<Types>(commands)...);
+  template <wpi::DecayedDerivedFrom<Command>... Commands>
```

Comment:
What's the difference between here and other places where this wasn't used (possibly `CommandScheduler.h` and `CommandPtr.h`)?

Reply:
It's equivalent to what was here before. I don't know why command-based decided to use CommandPtr in some places and derived classes of Command in others.

Reply:
CommandPtr was added later.  Maybe some places were missed in that add?

Reply:
The derived-rvalue-reference overloads were left in for backwards compatibility.